Contents/mods/Hydrocraft/media/scripts/Carpentry.txt

```diff
@@ -310,7 +302,7 @@ item HCWashtub
         DisplayName             = Wooden Washtub,
     ReplaceOnUseOn      = WaterSource-HCWashtubfull,
         icon            = HCWashtub,
-    
+		DisplayCategory	= CraftCarp,
```

Comment:
craftchem. technically can be a drainable containing water, but main use of this is for biodiesel only